.markdownlint.yml

```
@@ -87,7 +87,7 @@ MD032: true
 MD033: false
 
 # Bare URL used <https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md034>
-MD034: true
+MD034: false
```

```suggestion
MD034: true
```
Per my below comment you don't need to disable this rule.